exercises/01-revealing-module/color-generator.mjs

```diff
@@ -0,0 +1,14 @@
+const colors = ['red', 'green', 'blue'];
+
+let i = 0;
+
+export default function getNextColor() {
```

Comment:
In the exercise description we have "use default export" - what do you think about removing it? Or even replacing it with `named` exports? 

My initial idea was to reuse what is used in `Utilities.ts` file in HC Core and at the same time teach about "revealing-module-pattern". However, with es-modules in use - should we favor named exports?

Reply:
I think, that `named` exports would be the best approach and we should change this exercise description.